tests/testthat/test-basic.r

```diff
@@ -51,8 +51,13 @@ test_that("Correct method binding based on parameter types", {
     g(letters[1:3], stringName)
     g(rep(TRUE, 3), boolName)
     g(as.Date("2001-01-01") + 1:3, dateTimeName)
+
+    # test handling of S4 Objects
     g(c(testObj, testObj, testObj), objectName)
 
+    # test handling of external pointer to .NET objects
+    g(c(testObj@clrobj, testObj@clrobj, testObj@clrobj), objectName)
```

Comment:
A convenient place to test the referencing of external pointer SExpressions. The test above is identical but passes the S4.